lpc55_sign/src/debug_auth.rs

```diff
@@ -37,17 +39,21 @@ pub struct DebugAuthChallenge {
     challenge_vector: [u8; 32],
 }
 
-pub fn debug_credential(
+#[derive(Debug, Deserialize, Serialize)]
+pub struct DebugCredentialSigningRequest {
+    pub debug_public_key: RsaPublicKey,
+    #[serde(with = "SerHex::<StrictPfx>")]
```

Comment:
Can you provide a comment here with an example of how this field should be written in TOML? I've never heard of this `StrictPfx` thing.